cf-check/lmdump.c

```diff
@@ -92,7 +93,7 @@ int lmdump(lmdump_mode mode, const char *file)
     rc = mdb_env_create(&env);
     if (rc) return lmdump_report_error(rc);
 
-    rc = mdb_env_open(env, file, MDB_NOSUBDIR, 0644);
+    rc = mdb_env_open(env, file, MDB_NOSUBDIR | MDB_RDONLY, 0644);
```

Comment:
This looks a little unrelated to the commit.

Reply:
A necessary change, but maybe I should add a better message or add it as a separate commit.

Reply:
Put in a separate commit.